src/julia-syntax.scm

```diff
@@ -2766,7 +2767,8 @@
    '.>>>=   lower-update-op
 
    '|...|
-   (lambda (e) (error "\"...\" expression outside call"))
+   (lambda (e)
+	 (expand-tuple (cadr e)))
```

Comment:
```suggestion
	 (expand-tuple (cons 'tuple (cdr e))))
```

Reply:
That's what I tied at first, but when I do that I get
```julia
julia> (1,2)...
((1, 2),)
```
instead of the desired `(1, 2)`.

Reply:
Maybe `(list 'tuple e)`?